### PR TITLE
Blog updates Sept 26, 2024

### PR DESCRIPTION
**Pull request by stephenfuqua**
I did not create a ticket in the documentation or other projects. I just don't see it as necessary for blogging. There is no need to create a traceability path back to a ticket with requirements for a blog - at least, not as a general requirement. For a specific case it could be useful, like linking to a release ticket.

### CODE REVIEW

File: blog/2024-06-26.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: "📢 Milestone v0.1 has been reached"
```
Comment (stephenfuqua) [inline blog/2024-06-26.md:2]: The microphone emoji might be a bit much...

File: blog/2024-07-11.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: API Publisher v1.1.0
+authors: hoekstra
```
Comment (stephenfuqua) [inline blog/2024-07-11.md:3]: @jasonh-edfi please feel free to edit this with a pull request of your own!

File: blog/2024-09-20.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+title: API Publisher 1.2
+authors: [ hoekstra ]
+tags: [ release, api-publisher ]
+---
```
Comment (stephenfuqua) [inline blog/2024-09-20.md:5]: @jasonh-edfi another one that you can choose to edit.

File: blog/authors.yml [hunk 4]
```diff
@@ -1,23 +1,38 @@
...
-sfuqua:
+fuqua:
   name: Stephen A. Fuqua
   title: Director, Software Engineering
   url: https://www.ed-fi.org/success-stories/category/stephen-fuqua/
   image_url: https://www.ed-fi.org/assets/2020/10/Stephen-Fuqua-768x768.png
+  page: true
+  socials:
+    linkedin: stephenfuqua
+    github: stephenfuqua
```
Comment (stephenfuqua) [inline blog/authors.yml:23]: @vimayya @jasonh-edfi @MrEdData please feel free to submit a pull request adding social links like this.

File: blog/authors.yml [hunk 5]
```diff
@@ -1,23 +1,38 @@
...
   image_url: https://www.ed-fi.org/assets/2022/08/Mustafa-Yilmaz-July-2022-Headshot-768x768.png
+  page: true
+
+hunter:
+  name: Robert Hunter, Jr
+  title: Software Engineering
+
+ragone:
+  name: Maria Ragone
+  title: Technical Business Analyst
```
Comment (stephenfuqua) [inline blog/authors.yml:38]: @roberthunterjr (hm... Maria isn't in here yet...)

We'll add links to your headshots and www landing pages as soon as they are available. Welcome to add social links here if interested.

File: blog/2024-06-26.md (hunk 1 above)
Comment (roberthunterjr) [reply blog/2024-06-26.md:2]: I think its fine either way.

File: blog/authors.yml (hunk 5 above)
Comment (roberthunterjr) [reply blog/authors.yml:38]: Sounds good I will create a PR that adds this. Head shots are scheduled for the Wednesday following Summit.

**Review by roberthunterjr — APPROVED** [review]

**Pull request merged**